src/shape/custom_shapes.js

```diff
@@ -839,6 +894,10 @@ class Shape {
     this._splineProperties[key] = value;
   }
 
+  /**
+   * @method splineProperties
+   * @param  {Number} value
```

Comment:
I think this should be an object instead of a number, e.g. `splineProperties({ tightness: 0 })`

Reply:
oh actually we have this below already (line 1744) so we probably don't need this doc comment block

Reply:
Actually this block shouldn't even be there. The correct one has Object